app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+def create_report(data_file_name: str, report_file_name: str):
+    with open(data_file_name, 'r') as f:
+        with open(report_file_name, 'w') as wr_file:
+            buy = 0
+            supply = 0
+            for line in f.readlines():
+                if line != '':
+                    ls = line.split(',')
+                    if ls[0] == 'buy':
+                        buy += int(ls[1])
+                        continue
```

Comment:
Use `continue` here is not best practice because you have only 1 line after, use `else` instead